store/sqlstore/user_store.go

```diff
@@ -437,33 +437,29 @@ func (s SqlUserStore) GetEtagForProfiles(teamId string) store.StoreChannel {
 
 func (us SqlUserStore) GetProfiles(options *model.UserGetOptions) store.StoreChannel {
 	isPostgreSQL := us.DriverName() == model.DATABASE_DRIVER_POSTGRES
-	teamId := options.InTeamId
-	offset := options.Page * options.PerPage
-	limit := options.PerPage
+	return store.Do(func(result *store.StoreResult) {
+		query := usersQuery.
+			Join("TeamMembers tm ON ( tm.UserId = u.Id AND tm.DeleteAt = 0 )").
```

Comment:
Switching to a `JOIN` here from an additional `FROM` table. No semantic differences.